README.md

```diff
@@ -2,3 +2,33 @@
 Query balances for certain accounts and more on Ethereum.
 
 **This project is under development! Get back later for more information.**
+&nbsp;
+
+## Project dependencies
+
+#### With poetry:
+Run `poetry install` in the project root folder.
+#### With pip:
+Run `pip install -r requirements.txt`.
+&nbsp;
+
+## Using the cli tool:
+Inside the project root folder run `python balancy.py --help` for available commands.
+For some functions you will have to pass in a web3 provider url and a database uri as flags.
+If you have docker installed you can use `docker-compose up` to start an *etcd* database instance and a *ganache* as a web3 test node.
+Example commands with docker-compse running:
+```console
+python balancy.py addresses --w3url http://127.0.0.1:8545
```

Comment:
`python balancy.py api --w3url http://127.0.0.1:8545`
Start the API.

Reply:
I added the `--db-uri` flag too because the web api will also use a db connection.
`python balancy.py api --w3url http://127.0.0.1:8545 --db-uri http://127.0.0.1:2379`